dragonchain/lib/database/redisearch.py

```diff
@@ -186,7 +186,7 @@ def delete_index(index: str) -> None:
     try:
         client.drop_index()
     except redis.exceptions.ResponseError as e:
-        if str(e) != "Unknown Index name":  # Don't care if the index doesn't exist when trying to delete
+        if not str(e).startswith("Unknown Index name"):  # Don't care if the index doesn't exist when trying to delete
```

Comment:
man this sucks that we cant use a code or something

Reply:
I agree, but unfortunately the redisearch package doesn't handle errors at all, and just passes back the raw redis string.